packages/client/src/components/ratings/RatingBreakdownItems/RatingBreakdownItemsAlt.tsx

```diff
@@ -0,0 +1,41 @@
+import RectangularRatingIndicator from '@components/ratings/RectangularRatingIndicator'
+import { Rating } from '@futureproof/typings'
+import React from 'react'
+import { StyleSheet, View } from 'react-native'
+
+interface Props {
+  susRatings : number[]
+  susRatingsNames : string[]
+
+}
+
+const RatingBreakdownItems = ( { susRatings, susRatingsNames } : Props) => {
+  
+  const ratings = []
+  
+  for (let i = 0; i < susRatings.length; i++) {
+    ratings.push(
+      <RectangularRatingIndicator
+        key={i}
+        progressValue={susRatings[i]}
+        ratingName={susRatingsNames[i]}
+      />
+    )
+  }
```

Comment:
Changing the type of the props means that this can be changed to an inline map.

Reply:
Ok.